internal/app/init/internal/rootfs/mount/mount.go

```diff
@@ -237,6 +238,7 @@ func mountpoints() (mountpoints *mount.Points, err error) {
 		if dev, err = probe.GetDevWithFileSystemLabel(name); err != nil {
 			if name == constants.BootPartitionLabel {
 				// A bootloader is not always required.
+				log.Println("WARNING: no ESP partition was found")
```

Comment:
I found this useful for debugging. We might want to move this to a leveled log eventually.